tracer/src/Datadog.Tracer.Native/debugger_constants.h

```diff
@@ -36,6 +36,48 @@ const WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("ISymWrapper"),
                                 WStr("Datadog.Trace"),
                                 WStr("MSBuild")};
+
+const WSTRING general_error_message = WStr("Failed to instrument the method.");
+
+inline WSTRING GetGenericErrorMessageWithErrorCode(short errorCode)
+{
+    return general_error_message + WStr(" [Error Code: ") + shared::ToWSTRING(errorCode) + WStr("]");
+}
+
+const WSTRING invalid_probe_method_already_instrumented =
+    WStr("Dynamic Instrumentation failed to install the probe because the corresponding method is already instrumented by another product.");
+const WSTRING invalid_method_probe_probe_is_not_supported =
+    WStr("The method where the probe should have been placed is not supported for now.");
+const WSTRING line_probe_il_offset_lookup_failure =
+    WStr("There was a failure in determining the exact location where the line probe was supposed to be placed.");
+const WSTRING line_probe_in_async_generic_method_in_optimized_code =
+    WStr("Placing line probes in async generic methods in Release builds is currently not supported. Please note that this limitation only applies to optimized code.");
+const WSTRING invalid_probe_failed_to_instrument_method_probe = 
+    GetGenericErrorMessageWithErrorCode(1);
+const WSTRING invalid_probe_failed_to_instrument_line_probe =
+    GetGenericErrorMessageWithErrorCode(2);
+const WSTRING invalid_probe_probe_cctor_ctor_not_supported =
+    WStr("Instrumentation of constructors are not supported for now.");
+const WSTRING invalid_probe_probe_byreflike_return_not_supported =
+    WStr("Dynamic Instrumentation of methods that return a `ref struct` is not yet supported.");
+const WSTRING profiler_assemly_is_not_loaded =
+    GetGenericErrorMessageWithErrorCode(3);
+const WSTRING invalid_probe_failed_to_import_method_il =
+    GetGenericErrorMessageWithErrorCode(4);
+const WSTRING invalid_probe_failed_to_parse_locals =
+    GetGenericErrorMessageWithErrorCode(5);
+const WSTRING invalid_probe_failed_to_add_di_locals =
+    GetGenericErrorMessageWithErrorCode(6);
+const WSTRING invalid_probe_type_is_by_ref_like = WStr("Dynamic Instrumentation of methods in a `ref-struct` is not yet supported.");
```

Comment:
Nit - move line upwards for clarity

Reply:
Done in 7bdbd73b1bd67c0a2e90ac3c6adb3d7300cd819d.